### DISCUSSION TITLE
do-mpc still maintained?

### DISCUSSION THREAD

**Discussion by cahly** [do-mpc/do-mpc#165] [General]
Is do-mpc still being maintained? And what are the prospects of maintainence going forward?

**Comment by 4flixt** [comment]
Hi @cahly ,

yes do-mpc is still actively maintained! You are right however, there hasn't been an update to the main branch for ca. 7 month. We are currently implementing some cool new features on the develop branch and will hopefully be able to publish them soon. 

Is there any particular feature that you are waiting for? 

Best
4flixt

**Reply by cahly** [reply]
Hi @4flixt

That's good to hear, since it is a nice framework for my needs. Indeed I am waiting for a feature. It is the one discussed in #149. But more importantly i use do-mpc in a professional setting, which is probably ill advised as there are no guarantees, but good enough for now. We might choose a more commercial setup over time, but only if we have to.

Br Cahly
